src/ethereum/frontier/trie.py

```diff
@@ -26,6 +26,30 @@
 
 Node = Union[Account, Bytes, Transaction, Receipt, Uint, U256]
 
+from enum import Enum
```

Comment:
Looking good so far! I think you'll want to run the [linting tools](https://github.com/ethereum/eth1.0-specs#development), because there are a [few errors](https://github.com/ethereum/eth1.0-specs/pull/281/checks?check_run_id=3184303985#step:6:13).

Reply:
Im struggling to use the linting tools to solve the few errors i have. Ive installed Python 3.7 and Pypy 7.3 and unsure where to go from there.

Reply:
I was following your instruction and managed to install `tox` but at step 2 with `git submodule update --init --recursive` i got the error: 

> fatal: not a git repository (or any of the parent directories): .git

I continued to the second part (was unsure whether i had to complete the first part to proceed) and reached step 4. I didnt understand which directory to change to to install the tools.
Thanks for your help and please may you link the discord group because i cant seem to find it.

Reply:
You can find the Discord here: https://discord.com/invite/qGpsxSA

I have the same username, feel free to @ me in the `#specifications` channel and I can help you get set up!